config.toml

```diff
@@ -2,6 +2,8 @@ baseurl = "http://themes.gohugo.io"
 languageCode = "en-us"
 title = "Hugo Themes"
 
+[params]
+googleAnalytics = "UA-7131036-4"
```

Comment:
Please move `googleAnalytics` out of the `params` block. [`googleAnalytics`](https://github.com/spf13/hugo/blob/90d3fbf1da93a279cfe994a226ae82cf5441deab/tpl/tplimpl/template_embedded.go#L252) is site-wide variable like title.